Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -149,13 +149,12 @@ def audit
     end
 
     unless created_pr_comment
-      problem_count += new_formula_problem_lines.size
+      new_formula_problem_count += new_formula_problem_lines.size
       puts new_formula_problem_lines.map { |s| "  #{s}" }
     end
 
-    errors_summary = "#{Formatter.pluralize(problem_count, "problem")} in #{Formatter.pluralize(formula_count, "formula")}"
-    return if problem_count.zero?
-    ofail errors_summary unless strict
+    errors_summary = "#{Formatter.pluralize(problem_count + new_formula_problem_count, "problem")} in #{Formatter.pluralize(formula_count, "formula")}"
+    ofail errors_summary if problem_count > 0 || (new_formula_problem_count > 0 && !created_pr_comment)
```

Comment:
Can the above two lines get split into several lines each; there's a lot going on here that's hard to keep track of as-is.

Reply:
Could not split above line, Rubocop wants a guard clause in the last if statement. 
The condition is  ( if regular audits exist  or ( new formula audits exist but PR comment is not made ) ) 
then exit with non zero status